package.json

```diff
@@ -21,12 +21,12 @@
 		"release-zip": "npm run zip",
 		"pretest": "npm run build:production -- --display=none",
 		"pretest:visualtest": "VRT=1 npm run build:storybook",
-		"test": "npm run test:js:watch",
+		"test": "npm run test:js:watch -w ./tests/js",
 		"test:visualtest": "npm run test:visualtest -w ./tests/backstop",
 		"test:visualapprove": "npm run test:visualapprove -w ./tests/backstop",
 		"test:analyze": "npm run build:production -- --env.ANALYZE",
-		"test:js": "cross-env BABEL_ENV=test NODE_ENV=test jest --config=tests/js/jest.config.js --passWithNoTests",
-		"test:js:watch": "npm run test:js -- --watch",
+		"test:js": "npm run test:js -w ./tests/js",
+		"test:js:watch": "npm run test:js:watch -w ./tests/js",
 		"test:storybook": "npm run test:js -- --testMatch '<rootDir>/storybook/*.test.js'",
```

Comment:
Technically `test:storybook` is an E2E test (originally I thought it was JS but it uses Jest + Puppeteer) but looks like this may have been missed in putting it the E2E workspace.

Reply:
@aaemnnosttv I think this would be good option to move into `/storybook` workspace that is being introduced in #10091. WDYT?